src/app/components/listing/details/surveillance/surveillance-view.jsx

```diff
@@ -0,0 +1,185 @@
+              <TableCell
+                style={{ width: '33%' }}
+              >
+                Attribute
+              </TableCell>
+              <TableCell>Value</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <TableCell component="th" scope="row">
+                Date Surveillance Began
+                <ChplTooltip title="The date surveillance was initiated">
+                  <InfoOutlinedIcon
+                    className={classes.iconSpacing}
+                  />
+                </ChplTooltip>
+              </TableCell>
+              <TableCell>{ DateUtil.timestampToString(currentSurveillance.startDate, dateFormat) }</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableCell component="th" scope="row">
+                Date Surveillance Ended
+                <ChplTooltip title="The date surveillance was completed">
+                  <InfoOutlinedIcon
+                    className={classes.iconSpacing}
+                  />
+                </ChplTooltip>
+              </TableCell>
+              <TableCell>{ DateUtil.timestampToString(currentSurveillance.endDate, dateFormat) }</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableCell component="th" scope="row">
+                Surveillance Type
+                <ChplTooltip title="The type of surveillance conducted (either randomized or reactive). ONC-ACBs are required to randomly survey a minimum of 2% of all the Health IT modules to which they have issued an active certification.">
+                  <InfoOutlinedIcon
+                    className={classes.iconSpacing}
+                  />
+                </ChplTooltip>
+              </TableCell>
+              <TableCell>{ currentSurveillance.type.name }</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableCell component="th" scope="row">
+                Certification Criteria and Program Requirements Surveilled
+                <ChplTooltip title="The ONC Health IT Certification Program requirement that was surveilled. For example, this may be a specific certification criteria (e.g. 170.315(a)(1)), a transparency or disclosure requirement (e.g. 170.523(k)(1)), another requirement with a regulatory reference (e.g. 170.523(l)), or a brief description of the surveilled requirement.">
+                  <InfoOutlinedIcon
+                    className={classes.iconSpacing}
+                  />
+                </ChplTooltip>
+              </TableCell>
+              <TableCell data-testid="reqs-surveilled-cell">
+                { currentSurveillance.requirements?.length > 0
+                  && (
+                    <ul className={classes.unindentedData}>
+                      { currentSurveillance.requirements.map((req) => (
+                        <li key={req.id}>
+                          { `${req.type.name} ${req.criterion ? ': ' : ''}` }
```

Comment:
```suggestion
                          { `${req.type.name} ${req.criterion && ': '}` }
```

Not sure if this will work, but it would avoid having a ternary that has a `noop` else